test/e2e/tooltip.test.js

```diff
@@ -25,15 +25,15 @@ describe('vl-tooltip', async () => {
         await assert.eventually.equal(tooltip.isLeft(), location === 'left');
     }
 
-    async function beweegMuisOverButton(button) {
-        await button.hover();
+    async function hover(element) {
+        await element.hover();
```

Comment:
deze heeft dan zelfs nie veel nut meer denk ik?

Reply:
Ik dacht dat dat leesbaarder was (in de functie) dan 
```
await (await page.getElement()).hover();
```
maar het komt op hetzelfde neer.

Reply:
Ik vind onderstaande code persoonlijk leesbaarder (op it block niveau) dan een hulp functie:
```JavaScript
const button = await vlTooltipPage.getTopTooltipButton();
await button.hover();
```

Reply:
Ik ben (zoals je ondertussen al weet) ook geen fan van hulp functies om 'duplicatie' op it block niveau weg te werken, dat maakt (voor mij persoonlijk) een it block in isolatie lezen moeilijker.